.github/ISSUE_TEMPLATE/clarification-request.md

```diff
@@ -0,0 +1,74 @@
+---
+name: Clarification request
+about: You’ve found something confusing
+title: ''
+labels: docs
+assignees: ''
+
+---
+
+<!--
+  Feel free to delete anything that definitely does not apply to your
+  issue, as well as these comments. (Or don’t, it’s up to you.)
+
+  Please paste (a) link(s) from your browser showing the issue (bare
+  so it can be seen, not buried in a Markdown link).
+
+  Often a clarification request has as much to do with _where_ in the
+  docs something is addressed as _how._ If you think something is
+  useful but in the wrong place(s), or needs to be in multiple places
+  because of where different readers will look, say so!
+
+  If you're not sure where your request for clarification could be
+  best dealt with, either provide the URL where you think it should
+  be, provide multiple possible URLs, or delete the line.
+
+  Please don't provide a URL you wish existed that does not -- that
+  should be in a "New material" request instead.
+
+  -->
+  **Found in URL:** URL-HERE
+<!--
+  Also include the commit hash shown at the bottom of your browser
+  page, so we know what version you’re looking at. Better yet, include
+  the entire “From FILE.pod6 at perl6/doc on GitHub, commit a0b1c2d.”
+  line you’ll find as the first line of every doc page’s footer. -->
+**Found at commit:** COMMIT-OR-FROM-LINE-HERE
```

Comment:
Just why? The vast majority of issues are reported for the latest version of the docs (which is visible in the web version). Specifying the commit is an extra step that is not very useful.

Reply:
I’ve been reading all the docs issues for years—its the only perl6 repo whose issues I have diverted into my main inbox—and I tried to address issues that I felt came up with some frequency.

In this case, due to the lag from manual intervention between committing and presentation, I’ve seen the following happen all too often (and it’s very time wasting):
1. Person commits a change
2. Website gets rebuilt
3. Author notices an issue, often indexing or formatting (whether because they missed it on their private build or didn’t do one and saw it only once it went up on the site) and commits a fix
4. Time passes without a new build getting deployed
5. Somebody else posts an issue about the indexing or formatting issue
6. A third contributor, trying to be helpful, looks at the current source—not realizing it’s been corrected in master—and makes an unnecessary change to try to force the fix
7. Confusion ensues

So that’s the “just why”... if you think the cost-benefit analysis is in favor of the time saved by most issue submitters not copying and pasting a line from the footer over the well-meaning hyper-correctors’ time doing deckchair rearrangement, you’re probably less biased than me in making that call, since I’ve been on the pointy end of this particular stick a number of times and wished I’d known the submitter was looking at an old version of the site.

But that’s the why—I want to be sure that you’re aware of this and are making an informed cost-benefit call here rather than just looking for places to simplify.

Reply:
That does not happen so often, actually...

Reply:
> That does not happen so often, actually...

As you’re the one who has been doing the pushes, you may not have noticed how often it does happen. Looking at a git-blame of the omnibus docs (like control or operators) walking through a few levels deep, you’ll see how regressions and re-fixes (especially cargo-cultish ones involving indexing or unusual formatting corner cases) have happened in quick succession a few times a year.

Again, that may not be often enough to merit asking someone to mention the revision they noticed a problem in, but I, for one, would have noticed formatting or indexing issues (which are frequently hard to verify from inspection) had already been fixed in master when the issue was filed at least twice in recent memory if I saw it was a week-old revision and checked if the current one still had the issue. (In one case, I even went so far as committing the effective null-op, thinking I had fixed something, until I went to try to fix the “root cause” and discovered the “underlying issue” didn’t exist.)

If we always built the docs on the current rev whenever an issue came in, so we could see it for ourselves, that would be different. But sticking a cycle of building the docs at one revision, then making changes and rebuilding them again, is a lot of friction until incremental updating is easy.

(In another ticket #2668 I see that it may soon be easy; if that’s the case, then once it’s deployed and documented, I don’t think asking for a revision where the issue was noticed would be necessary anymore. If that’s soon, then I’ll just remove this field and resolve.)